### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_ReactLogin_NonTextContrast: The color contrast ratio of the focus indicator of the "Login" button and "Remember me" checkbox is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42370]
![42370_A11y_ NET Core_ASP NETCore_ReactLogin_NonTextContrast](https://user-images.githubusercontent.com/86710491/175495520-80539fbc-6b1b-4883-93ad-d5c864053f8d.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir React"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new react -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Test the Login Page
10. Turn on Color contrast analyzer and check that The Color contrast ratio of the focus indicator of the "Login" button and "Remember me" checkbox is less than 3:1.

### Actual:
The Color contrast ratio  the focus indicator of the "Login" button and "Remember me" checkbox is less than 3:1.
**This issue is observed throughout the application for all the "Login" buttons and "Remember me" checkboxes.**

### Expected:
The Color contrast ratio of the focus indicator of the "Login" button and "Remember me" checkbox should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/spa-templates/pull/73

**Comment by Truptik12** [comment]
@HaoK 

We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version and the bug.
![image](https://user-images.githubusercontent.com/86710491/185584623-e080029c-b2b4-43cf-8f49-958491ff34c3.png)
Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.

**Comment by Truptik12** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.


![image](https://user-images.githubusercontent.com/86710491/188160201-ca088e8c-0aae-442e-8e3d-e1d1636397eb.png)

**Comment by merriemcgaw** [comment]
@Truptik12 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached.

![image](https://user-images.githubusercontent.com/86710491/192524165-36f5bbeb-2e82-460f-b8f1-9815d0083115.png)

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
